<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WriteLetterWindow</class>
 <widget class="QMainWindow" name="WriteLetterWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>980</width>
    <height>712</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="Lcontext">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>240</y>
      <width>811</width>
      <height>331</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>101</width>
      <height>181</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="Lreciver">
       <property name="text">
        <string>收件人</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="LreciverOther">
       <property name="text">
        <string>抄送人</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="LreciverSecert">
       <property name="text">
        <string>密送人</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="Ltheme">
       <property name="text">
        <string>主题</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>20</y>
      <width>711</width>
      <height>412</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0,0,0">
     <property name="spacing">
      <number>20</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="4,3,2">
       <property name="spacing">
        <number>90</number>
       </property>
       <item>
        <widget class="QTextEdit" name="Lreceiver"/>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ReceiveListButton">
         <property name="text">
          <string>联系人列表</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTextEdit" name="LreceivOther"/>
     </item>
     <item>
      <widget class="QTextEdit" name="LreceiverSecert"/>
     </item>
     <item>
      <widget class="QTextEdit" name="LthemInput"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>200</y>
      <width>811</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,1,5,1">
     <item>
      <widget class="QPushButton" name="LchooseFont">
       <property name="text">
        <string>选择全文字体</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="LchooseFontColor">
       <property name="text">
        <string>选择全文颜色</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="SaveDraftButton">
       <property name="text">
        <string>存至草稿箱</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>570</y>
      <width>811</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,4,1">
     <item>
      <widget class="QPushButton" name="UPloadattach">
       <property name="text">
        <string>附件上传</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Lsend">
       <property name="text">
        <string>发送</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
